components/host-dashboard/CreatePendingOrderModal.tsx

```diff
@@ -139,7 +139,6 @@ const createPendingContributionModalCollectiveQuery = gql`
       ... on AccountWithContributions {
         tiers {
           nodes {
-            id
```

Comment:
Noticed this duplication while doing this, so thought might as well remove it.

Reply:
Good move. It means ESLint might not be running  properly for GQL in TSX files, since it's supposed to detect this kind of mistakes